File: Beecrowd/1158 Soma de Ímpares Consecutivos III/main.py
soma = 0
cont = 0
N = int(input())

for i in range(N):
  entrada = [int(a) for a in input().split()]
  X = entrada[0]
  Y = entrada[1]
  j = X
  while(cont < Y):
    if j % 2 != 0:
      soma += j
      cont += 1
    j += 1     
  print(soma)
  soma = 0
  cont = 0

File: Beecrowd/1158 Soma de Ímpares Consecutivos III/main.cpp
#include <iostream>
 
using namespace std;

int main(){
  int N, X, Y, i, j, soma = 0, cont;

  cin >> N;

  for (i = 0; i < N; i++){
    cin >> X >> Y;
    for(j = X, cont = 0; cont < Y; j++)
      if(j % 2 != 0){
        soma += j;
        cont++;
      }
    cout << soma << endl;
    soma = 0;
  }
  return 0;
}